### ISSUE TITLE
Resurrection of Sitobolium Desv. (Dennstaedtiaceae) [PASSED]

### ISSUE THREAD

**Issue by GuguYanez** [pteridogroup/ppg#6]
### Name of taxon

Sitobolium

### Rank of taxon

Genus

### Approximate number of species affected

five. Sitobolium appendiculatum (Wall. ex Hook.) L.A.Triana & Sundue, Sitobolium hirsutum (Sw.) L.A.Triana & Sundue, Sitobolium punctilobulum (Michx.) Desv., Sitobolium wilfordii (T.Moore) L.A.Triana & Sundue, Sitobolium zeylanicum (Sw.) L.A.Triana & Sundue. 

### Description of change

Resurrection of Sitobolium Desv. (Dennstaedtiaceae)

### Reason for change

This genus of ferns was resurrected by Triana-Moreno et al. (2022b) after performing a phylogeny of the subfamily Dennstaedtioideae (Dennstaedtiaceae) based on four regions of the chloroplast (rbcL coding gene, rpl16 intron, rps4 coding region along with the rps4-trnS intergenic spacer, and the trnL-F intergenic spacer). In the article the authors carried out several taxonomic changes according to the phylogenetic relationships suggested by morphological and molecular evidence and several autapomorphic states recovered through ancestral character state reconstruction. 

The authors proposed a classification that recognizes four morphologically diagnosable and geographically coherent Dennstaedtioideae clades as genera: Dennstaedtia that consists of the bulk of species traditionally recognized in that genus; Mucura, which includes D. globulifera and D. bipinnata; Sitobolium, which  includes a set of north-temperate species, and Microlepia that includes the species traditionally recognized in that genus. This treatment is made possible by an associated proposal to conserve Dennstaedtia with a new type (D. cicutaria; Triana-Moreno & al., 2022a). (While it’s beyond the scope of this PPG II proposal, conservation of Dennstaedtia with a new type was necessary in order to avoid the large number of name changes that would be associated with transferring Microlepia species to Dennstaedtia and the bulk of Dennstaedtia to Patania).

Sitobolium was in use by early authors until Moore (1859) subsumed it under his concept of Dennstaedtia (Tryon & Tryon, 1980). In Triana-Moreno et al. (2022b) Sitobolium constitutes a small clade of ca. five species sister to Microlepia. The species of the genus are distinguished by their relatively small leaves that have elongate catenate hairs, epipetiolar buds, enlarged vein endings, and marginal cup-shaped sori comprised of both abaxial and adaxial indusia. The spores are verrucate with prominent ridges, except in Sitobolium appendiculatum (ex Emodiopteris) and S. wilfordii (ex Coptidipteris) in which they are tuberculate, but otherwise lack diagnostic characters to distinguish them as a distinct genus and are therefore included in Sitobolium.

### Reference(s) for publication of the name

Triana-Moreno, L.A., Schwartsburd, P.B., Yañez, A., Pena, N.T.L., Kuo, L.Y., Rothfels, C. & Sundue, M. 2022a. (2892) Proposal to conserve the name Dennstaedtia (Dennstaedtiaceae) with a conserved type. Taxon 71: 688–690. https://doi.org/10.1002/tax.12756 

Triana‐Moreno, L. A., Yañez, A., Kuo, L. Y., Rothfels, C. J., Pena, N. T. L., Schwartsburd, P. B., & Sundue, M. (2022b). Phylogenetic revision of Dennstaedtioideae (Dennstaedtiaceae: Polypodiales) with description of Mucura, gen. nov. TAXON 72(1): 20–46. https://doi.org/10.1002/tax.12858

### List the numbers of any related issues

#4

### Code of Conduct

- [X] I agree to follow the PPG Code of Conduct

**Comment by alanweakley** [comment]
I support this proposal.  I am already following it in the Flora of the Southeastern US Project:  https://fsus.ncbg.unc.edu/show-taxon.php?&parentid=66186&limit=1&offset=4&taxonid=67971

**Comment by schwartsburd** [comment]
Dennstaedtia (sensu Kramer & Green 1990, PPG I 2016) is polyphyletic. The new scheme provided by Triana-Moreno & al. (I participated as co-author) makes Dennstaedtia s.str. monophyletic, rescues Sitobolium, re-defines Microlepia, and created a new genus, Mucura, to accomodate two Neotropical species. All genera can be morphologically recognizable, and this new classification is monophyletic. It also minimized new combinations, since we proposed some type conservations. I strongly agree with this proposal, and that of Mucura.
